dockers/docker-snmp/snmpd.conf.j2

```diff
@@ -13,12 +13,38 @@
 #  AGENT BEHAVIOUR
 #
 
-#  Listen for connections on all ip addresses, including eth0, ipv4 lo
+#  Listen for connections on ip addresses based on configuration
 #
+# if SNMP_AGENT_ADDRESS_CONFIG is present
+# use SNMP_AGENT_ADDRESS_CONFIG
+# elif either MGMT_INTERFACE or LOOPBACK_INTERFACE
+# use management and loopback ip addresses
+# use docker0 ipv4 and ipv6 address
+# else if none of the above config is present
+# listen on any IP
```

Comment:
The logic is complex.
how about minigraph.py just generated correct SNMP_AGENT_ADDRESS_CONFIG?
1. single-asic, bind to mgmt and lo
2. multi-asic bind to docker0

Reply:
I think docker0 IP is not part of user config, you can use template here.

However, mgmt IP, lo IP are from user config, it is natural to add them into ConfigDB SNMP_AGENT_ADDRESS_CONFIG table.

Reply:
I see the challenge of determine single/multi-asic in minigraph.py. So it is okay to keep complex code here.

Reply:
I have modified the snmpd.conf.j2 template to use management and loopback IP for single asic platform.
With this, we could backport to previous branches to ensure that we could use ipv6 .

Next step is to:
1. If the change to modify to use bridge network for snmp docker is merged in, we could add NAT rule for single asic platform similar to multi-asic platform.
2. After that change, will update minigraph parser to update SNMP_AGENT_ADDRESS_CONFIG table with management IP and docker0 IP for both single and multi-asic platform that way minigraph parser need not have the logic to differentiate between single and multi-asic platform.
3. Remove the changes in snmpd.conf.j2 and directly use SNMP_AGENT_ADDRESS_CONFIG which will be updated by minigraph parser.